controllers/machinehealthcheck_controller.go

```diff
@@ -632,5 +635,38 @@ func (r *MachineHealthCheckReconciler) externalRemediationRequestExists(ctx cont
 	if err != nil {
 		return false
 	}
+	logger := ctrl.LoggerFrom(ctx)
+	if r.isAlertOldEmr(remediationReq, logger) {
+		//TODO mshitrit send alert here
+		//metrics.ObserveMachineHealthCheckOldEmr(m.Name, m.Namespace)
+	}
 	return remediationReq != nil
 }
+
+func (r *MachineHealthCheckReconciler) isAlertOldEmr(emr *unstructured.Unstructured, logger logr.Logger) bool {
+	//verify emr exist
+	if emr == nil {
+		return false
+	}
+	isSendAlert := false
+	//verify emr is old
+	if time.Now().After(emr.GetCreationTimestamp().Add(oldEmrAlertTimeout)) {
```

Comment:
Might be I'm wrong, but I did not find any note about old EMR in the [MHC proposal](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20191030-machine-health-checking.md#external-remediation).
Updating the document should help all the EMR implementers to understand what we are trying to achieve and take befit from this feature